assets/js/modules/adsense/datastore/clients.js

```diff
@@ -155,6 +157,40 @@ const baseSelectors = {
 
 		return clients[ accountID ];
 	},
+
+	/**
+	 * Gets the AdSense For Content (AFC) client for the given AdSense account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state     Data store's state.
+	 * @param {string} accountID The AdSense Account ID to fetch AFC client for.
+	 * @return {(Object|undefined)} An AdSense client; `undefined` if not loaded.
```

Comment:
See my other comment:
```suggestion
	 * @return {(Object|null|undefined)} An AdSense AFC client, or `null` if none exists; `undefined` if not loaded.
```

Reply:
👍